conformance/local_symtab.ion

```diff
@@ -2,70 +2,80 @@
 // SPDX-License-Identifier: Apache-2.0
 
 
-// Non-structs pass through
-(ion_1_x
+(ion_1_x "Non-structs pass through"
   (then (toplevel first $ion_symbol_table::[] second)
         (produces first $ion_symbol_table::[] second))
   (then (toplevel first $ion_symbol_table::() second)
         (produces first $ion_symbol_table::() second)))
 
-// IST structs are elided from app view
-(ion_1_x
+
+(ion_1_x "IST structs are elided from app view"
   (then (toplevel first $ion_symbol_table::{} $ion_symbol_table::null.struct second)
         (produces first                                                      second))
-  // ...but only at top-level
-  (then (toplevel [first, $ion_symbol_table::{}, $ion_symbol_table::null.struct, second])
+  (then "but only at top-level"
+        (toplevel [first, $ion_symbol_table::{}, $ion_symbol_table::null.struct, second])
         (produces [first, $ion_symbol_table::{}, $ion_symbol_table::null.struct, second])))
 
 
-// Basic symbol expansion
-(ion_1_0
-  (each (toplevel $ion_symbol_table::{symbols:["a"]})
-        (toplevel             '#$3'::{  '#$7':["a"]})
-        (then (toplevel '#$10'::'#$10' '#$10')
-              (produces      a::a         a))))
-
-
-// Out-of-bounds SID
-//  At present, the DSL doesn't support partial expansion of the front of stream.
-(ion_1_0
-  (then (toplevel '#$10')
-        (signals "Invalid symbol address: 10"))
-  (then (toplevel $ion_symbol_table::{symbols:["a"]})
-        (then (toplevel '#$11')
-              (signals "Invalid symbol address: 11")))
-  (then (toplevel $ion_symbol_table::{symbols:["a", "b"]})
-        (then (toplevel '#$10' '#$11')
-              (produces a b))
-        (then (toplevel '#$12')
-              (signals "Invalid symbol address: 12"))))
-
-
-// Empty, null, or malformed `symbols` field is ignored
-(ion_1_0 (each (toplevel $ion_symbol_table::{symbols:[]       })
-               (toplevel $ion_symbol_table::{symbols:null.list})
-               (toplevel $ion_symbol_table::{symbols:42       })
-               (toplevel $ion_symbol_table::{symbols:("b")    })
-               (then (toplevel '#$10')
-                     (signals "Invalid symbol address: 10"))))
-
-
-// Non-string and null (local) `symbols` entries act like $0
-(ion_1_0
-  (then (toplevel $ion_symbol_table::{symbols:[a, 1, true, ()]}
+(ion_1_0 "User symbols expand as expected"
+  (then "with one-item local symtab"
+        (each (toplevel $ion_symbol_table::{symbols:["a"]})
+              (toplevel             '#$3'::{  '#$7':["a"]})
+              (toplevel '#$10'::'#$10' '#$10')
+              (produces      a::a         a)))
```

Comment:
The `(toplevel '#$10'::'#$10' '#$10')` should be in a `then` clause, right?
```suggestion
              (then (toplevel '#$10'::'#$10' '#$10')
                    (produces      a::a         a))))
```

Reply:
Yes, thank you.